ansible_collections/arista/avd/roles/eos_designs/tasks/main.yml

```diff
@@ -29,6 +29,9 @@
     avd_switch_facts: true
     #cprofile_file: "eos_designs_facts.prof"
     template_output: true
+    schema: "{{ lookup('ansible.builtin.file', role_schema_path) | from_yaml }}"
```

Comment:
so I am not sure where `role_schema_path` is being set in this PR

Reply:
it is already set under `vars/main.yml`